### PR TITLE
Use python3 for post SLE12 and kubic as image name for Factory

### PR DESCRIPTION
**Pull request by thkukuk**

### CODE REVIEW

**Comment by thkukuk** [comment]
On Thu, Apr 12, Maximilian Meister wrote:

> @MaximilianMeister approved this pull request.
> 
> LGTM
> 
> if it has to do with https://bugzilla.suse.com/show_bug.cgi?id=1085632 you
> could add the bsc# to the commit message

No, it hasn't.

  Thorsten

-- 
Thorsten Kukuk, Distinguished Engineer, Senior Architect SLES & CaaSP
SUSE LINUX GmbH, Maxfeldstr. 5, 90409 Nuernberg, Germany
GF: Felix Imendoerffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nuernberg)

File: packaging/suse/make_spec.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ cat <<EOF > ${NAME}.spec
 %endif
 
 %if 0%{?is_opensuse} && 0%{?suse_version} > 1500
-  %define _base_image tumbleweed
+  %define _base_image kubic
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply packaging/suse/make_spec.sh]: I see it is in the title name. LGTM

File: packaging/suse/make_spec.sh [hunk 2]
```diff
@@ -64,8 +64,17 @@ Url:            https://github.com/kubic-project/salt
...
+%if 0%{?suse_version} >= 1500
+Requires:       python3-M2Crypto
+Requires:       python3-pyOpenSSL
+Requires:       python3-etcd
+Requires:       python3-pyroute2
+%else
 Requires:       python-m2crypto
 Requires:       python-pyOpenSSL
+Requires:       python-etcd
+Requires:       python-pyroute2
```
Comment (MaximilianMeister) [inline packaging/suse/make_spec.sh]: @jordimassaguerpla JFYI we don't have this package in d:C:H:CN yet

**Comment by kiall** [comment]
Looks like CI is failing due to what we believe is an issue with GitHub... (@ereslibre has hit this once or twice yesterday).

PRs which start in a external repo like this one seem to not clone correctly!

	kiall@kiall-z620:/tmp> git clone https://github.com/kubic-project/salt test-salt
	Cloning into 'test-salt'...
	remote: Counting objects: 5062, done.
	remote: Compressing objects: 100% (205/205), done.
	remote: Total 5062 (delta 226), reused 297 (delta 181), pack-reused 4673
	Receiving objects: 100% (5062/5062), 2.47 MiB | 552.00 KiB/s, done.
	Resolving deltas: 100% (3056/3056), done.

	kiall@kiall-z620:/tmp> cd test-salt/

	kiall@kiall-z620:/tmp/test-salt> git fetch origin pull/475/head:pull-475
	remote: Counting objects: 5, done.
	remote: Compressing objects: 100% (2/2), done.
	remote: Total 5 (delta 2), reused 5 (delta 2), pack-reused 0
	Unpacking objects: 100% (5/5), done.
	From https://github.com/kubic-project/salt
	 * [new ref]         refs/pull/475/head -> pull-475
	kiall@kiall-z620:/tmp/test-salt> git checkout pull-475 
	Switched to branch 'pull-475'

	kiall@kiall-z620:/tmp/test-salt> git log -1
	commit 026ea396b13d494b58925aa3803e6279cba88531 (HEAD -> pull-475)
	Author: Thorsten Kukuk <kukuk@thkukuk.de>
	Date:   Thu Apr 12 15:22:56 2018 +0200

	    Use python3 for post SLE12 and kubic as image name for Factory

In this case - the second commit is missing from the checkout! I'll see if theres some way to report this to @github

**Comment by kiall** [comment]
Since we have other PRs which reproduce the GitHub issue (https://github.com/kubic-project/salt/pull/479 / https://github.com/kubic-project/salt/pull/480 / https://github.com/kubic-project/salt/pull/481 ) - will merge this.

**Pull request merged**